test/e2e-go/cli/goal/expect/goalNodeTest.exp

```diff
@@ -79,7 +79,7 @@ if { [catch {
     spawn goal node start -d $TEST_PRIMARY_NODE_DIR
     expect {
         timeout { close; ::AlgorandGoal::Abort "starting node exceeded timeout" }
-        -re {^Algorand node failed to start: node exited with an error code, check node\.log for more details : exit status 1} {
+        -re {^Algorand node failed to start: node exited with an error code, check node\.log for more details : exit status .*} {
```

Comment:
yes, that's my thinking as well - but maybe `/d*` to ensure it's digits.

Reply:
done